cucumber/api/features/policy_versions_retention.feature

```diff
@@ -24,3 +24,14 @@ Feature: Limit the number of policy versions
     """
     Deleting policy version: {:version=>1, :resource_id=>"cucumber:policy:policy_test_version",
     """
+
+  Scenario: Migrate db remove extra policies
+    Given I save my place in the log file
+    And I successfully POST 21 times "/policies/cucumber/policy/policy_test_version" with body:
+    """
+      - !user bob
+    """    
+    And the HTTP response status code is 201
+    When I migrate the db
+    And I successfully GET "/resources/cucumber/policy/policy_test_version"
+    Then there are 20 "policy_versions" in the response
```

Comment:
Please add some logs verification to the test to understand the job was done

Reply:
As far as I know-  upgrade (db migrate) does not written to the log in a testable matter, can you elaborate on how do you want to test it?

Reply:
In our sanity we have something like this
`    And The following appears in the log after my savepoint:
    """
    Deleting policy version: {:version=>1, :resource_id=>"cucumber:policy:policy_test_version",
    """`
Something like this, as part of the upgrade we dont want to see this info on our logs ?

Reply:
no, because we're running just a SQL script for multiple deletions

Reply:
@InbalZilberman WDYT? during the migrate do we want/need to have some logs?